airbyte-integrations/connectors/source-zendesk-talk/metadata.yaml

```diff
@@ -31,5 +31,5 @@ data:
   supportLevel: certified
```

Comment:
@ambirdsall if this change is breaking, you'd have to add the breaking change fields here.
```suggestion
  supportLevel: certified
  releases:
    breakingChanges:
      0.2.0:
        upgradeDeadline: 2024-04-01
        message: "Incremental stream state format has changed in this version."
```

/cc @katmarkham and @ChristoGrab — do we include the detailed breaking change message straight in yaml? I thought it had additional key, right?

Reply:
The suggested change is accurate, all thats needed here is the deadline and short blurb. We also need to create a "zendesk-talk-migrations.md" file which will contain the more detailed breakdown of the changes and the migration steps, I've updated the migration template with the info on setting this up: https://docs.google.com/document/d/1SR0S7Xhv2HavV3lFdyzTdk_vWnXJX7XJwv5OkWExOvc/edit

If anything is unclear let me know!